client/src/components/share/share-template.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import type { ShareRedirectProps } from './types';
+
+export const ShareTemplate = ({
+  handleRedirectToTwitter
+}: Pick<ShareRedirectProps, 'handleRedirectToTwitter'>): JSX.Element => {
+  return (
+    <button
+      data-testid='ShareTemplateWrapperTestID'
+      className='btn fade-in'
+      onClick={handleRedirectToTwitter}
+    >
+      <FontAwesomeIcon title='twitterIcon' icon={faTwitter} size='1x' />
```

Comment:
```suggestion
      <FontAwesomeIcon icon={faTwitter} size='1x' aria-hidden="true"/>
```

The icon should be hidden from screen reader users. We can use either sr-only text or `aria-label` to provide an accessible name for this which screen reader users will hear.

Reply:
Thanks for keeping your time for adding suggestions